addons/pos_event/static/src/app/screens/receipt_screen/receipt_screen.js

```diff
@@ -19,7 +19,35 @@ patch(ReceiptScreen.prototype, {
         ]);
     },
     async printEventBadge() {
-        const registrations = this.pos.get_order().eventRegistrations.map((reg) => reg.id);
-        await this.report.doAction("event.action_report_event_registration_badge", [registrations]);
```

Comment:
In the old code, the registration IDs were passing wrapped in an array as `[registrations]`. To double check it was okay to change this, I had a look at why this was working. It turns out the IDs are eventually passed to `.join` to make a string, and JS flattens arrays in the process:
![image](https://github.com/user-attachments/assets/3d170f0c-25eb-4033-817a-f9d2d9acbc06)

So I think it's fine to change this and makes it more correct even if it happened to not affect anything before.